bids-validator/bids_validator/rules/file_level_rules.json

```
@@ -377,7 +377,9 @@
         "_channels\\.tsv",
         "_eeg\\.json",
         "_coordsystem\\.json",
-        "_photo\\.jpg"
+        "_photo\\.jpg",
+        "_photo\\.png",
+        "_photo\\.tif"
```

```suggestion
        "_photo\\.jpg"
```

The change in "photo" extensions only applies to "microscopy" modality, so not here for "eeg".
Sorry that was not clear in the list.

---

Done